site/src/sphinx/client-timeouts.rst

```diff
@@ -0,0 +1,47 @@
+.. _client-timeouts:
+
+Overriding client timeouts
+==========================
+Sometimes, the default timeouts used by the Armeria-generated client does not suit a particular scenario well. In these cases, you might want to override the timeout settings.
```

Comment:
Should we also document the system properties that can control the defaults? Especially for the case where debugging in IntelliJ or something, using system properties in the debug configuration can be even simpler than changing the code.

Reply:
For reference, the properties are documented around here

https://line.github.io/armeria/apidocs/com/linecorp/armeria/common/Flags.html#defaultRequestTimeoutMillis()

Reply:
Let me send a follow-up pull request after merging this one.